feat(Masthead): Updates for new masthead structure in Penta (#10809)
patternfly/patternfly-react: packages/react-core/src/components/Masthead/examples/Masthead.md

id: Masthead
section: components
cssPrefix: pf-v5-c-masthead
propComponents: ['Masthead', 'MastheadToggle', 'MastheadMain', 'MastheadBrand', MastheadLogo, 'MastheadContent']
---

import BarsIcon from '@patternfly/react-icons/dist/js/icons/bars-icon';
import { Link } from '@reach/router';
import pfIcon from '../../assets/PF-HorizontalLogo-Color.svg';

> `Masthead` should contain the following components to maintain proper layout and formatting: `MastheadMain`, and `MastheadContent`.

Reviewer: ```suggestion
To maintain proper layout and formatting, a `<Masthead>` should contain both a `<MastheadMain>` and `<MastheadContent>` component.
```

> `MastheadMain` represents the smaller area taken up by a logo, and will typically contain a `MastheadLogo`. `MastheadContent` represents the main portion of the masthead area and will typically contain a `Toolbar` or other menu-like content such as `Dropdown`.

Author: Yes!  I meant to go back an look at the wording.  I will see if Erin can review it too.

> `MastheadMain` represents the smaller area taken up by a toggle and logo, and will typically contain a `MastheadToggle` and a `MastheadLogo`. `MastheadContent` represents the main portion of the masthead area and will typically contain a `Toolbar` or other menu-like content such as `Dropdown`.

Reviewer: ```suggestion
The `<MastheadMain>` component includes the smaller area that typically contains the `<MastheadToggle>` and a `<MastheadLogo>`. `<MastheadContent>` represents the main portion of the masthead, and will typically contain a `<Toolbar>` or other menu-like components, like a `<Dropdown>`.
```
